### PR TITLE
[TW-1064] Clarify "Validating APIs" page is for syntax validation

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Changed "Validating APIs" page to "Checking API syntax and rules"
- Removed references to "validation" when discussing checking API definition syntax
- Removed outdated "Validating elements" section
- Moved "Viewing rule violations in your API definition" to "Checking API syntax and rules" page

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md [hunk 1]
```diff
@@ -29,96 +29,66 @@ contextual_links:
...
 ![API definition error](https://assets.postman.com/postman-docs/v10/schema-validation-error-open-v10.jpg)
 
-> Note that sometimes a single error in your definition will cause more than one issue to appear in the list. As you fix your errors, the validation issues disappear.
+## Viewing rule violations in your API definition
 
-Postman will display a warning if there is an issue with your definition JSON or YAML syntax. Look for errors indicated in the editor and hover over them for more detail. For Postman to be able to validate your definition elements, the JSON or YAML must be well-formed.
+> [This feature is available on Postman Enterprise plans.](https://www.postman.com/pricing)
 
-If there are no errors, Postman will indicate in the lower pane that your API definition is valid.
+As you create your API definition in the editor, Postman automatically checks it against the [Postman API Governance and API Security](/docs/api-governance/api-governance-overview/) rules configured for your team. Postman displays any rule violations below the editor. Resolving these issues enables you improve your API definition.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md:55]:
```suggestion
As you create your API definition in the editor, Postman automatically checks it against the [Postman API Governance and API Security](/docs/api-governance/api-governance-overview/) rules configured for your team. Postman displays any rule violations below the editor. Resolving these issues enables you to improve your API definition.
```

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md [hunk 2]
```diff
@@ -29,96 +29,66 @@ contextual_links:
...
+To hide a rule violation for the API definition, do the following:
 
-If you want your requests to be validated, your collection must be linked to an API.
+1. Select **Hide** next to the rule violation.
 
-A collection is linked to an API if you [generate it from a schema](/docs/designing-and-developing-your-api/developing-an-api/defining-an-api/#generating-an-api-definition) or [add it as a relation to an existing API](/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements/#adding-a-collection).
+    <img alt="Hide a rule violation in your API definition" src="https://assets.postman.com/postman-docs/v10/api-definition-hide-rule-violation-v10.1.jpg"/>
 
-Postman validates a request when it's sent. If there are issues with the request, Postman displays a warning message showing the number of issues next to the name of the request.
+1. Select a reason that you want to hide it, then select **Hide** again.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md:79]: What do you think about removing "again" from the sentence? To me, using "again" 4 implies that you're selecting the same button/link a second time.

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md [hunk 3]
```diff
@@ -29,96 +29,66 @@ contextual_links:
...
+When you make updates to your API definition, Postman re-checks it. If your changes resolve the issue, Postman removes the rule violation from the list.
 
-![Test validation](https://assets.postman.com/postman-docs/api-builder-test-validation.jpg)
+### Hiding rule violations
 
-### Validating requests
+To hide a rule violation for the API definition, do the following:
 
-If you want your requests to be validated, your collection must be linked to an API.
+1. Select **Hide** next to the rule violation.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md:75]: What do you think about adding the icon that appears in the button? For example:

`1. Select Unmask variable icon **Hide** next to the rule violation.`

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md:79]: Totally agree with you!

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/validating-elements-against-schema.md:75]: Great idea!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by shashankawasthi88 — APPROVED** [review]
Looks good

**Pull request merged**